pkgs/by-name/no/notonoto/package.nix

```
@@ -2,7 +2,10 @@
   lib,
   fetchFromGitHub,
   stdenvNoCC,
+  fontforge,
+  python3Packages,
```

We just need regular python3 here instead of python3Packages
```suggestion
  python3,
```